rofi-pass

```diff
@@ -22,6 +22,7 @@ OTPmethod_field='otp_method'
 default_autotype="user :tab pass"
 delay=2
 wait=0.2
+xdotool_delay=12
```

Comment:
Can xdotool reports its own delay? I'd rather we used `: ${xdotool_delay:-"$(xdotool get delay)"}` or something along these lines.

Reply:
xdotool's delay is always 12, if not specified